india_compliance/gst_india/overrides/transaction.py

```diff
@@ -422,20 +422,6 @@ def _throw(message, title=None):
     return all_valid_accounts
 
 
-def validate_tax_accounts_for_non_gst(doc):
```

Comment:
at the time of setting GST details,
check => Non-GST with Tax Rate/Amount Throw.